lib/screens/global_search/search_bloc/search_bloc.dart

```diff
@@ -10,15 +10,12 @@ import 'package:rxdart/rxdart.dart';
 import 'bloc.dart';
 
 class SearchBloc extends Bloc<SearchEvent, SearchState> {
-  SearchBloc(this.searchRepo);
+  SearchBloc(this.searchRepo) : super(InitialSearchState());
```

Comment:
This is nice :)

Reply:
I liked overriden property more, but this is safer as it requires you to provide default value